lib/onelogin/ruby-saml/authrequest.rb

```diff
@@ -28,6 +28,8 @@ def create(settings, params = {})
 
       def create_params(settings, params={})
         params = {} if params.nil?
+        # Some ruby-saml versions uses :RelayState others use 'RelayState'
```

Comment:
@Lordnibbler that is true when someone is using Rails, but this gem doesn't require or use Rails by default, so I would think the usage internally here should be consistent. Whichever method this gem chooses will then work fine since Rails will support either string or symbol keys. Am I missing something?